certificate/src/main/java/com/epam/esm/dto/PaginationParameter.java

```diff
@@ -4,33 +4,31 @@
 
 public class PaginationParameter {
```

Comment:
I'd suggest to use [Pageable](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/Pageable.html) interface instead.